packages/vue/package.json

```diff
@@ -53,7 +53,6 @@
     "@vitejs/plugin-vue": "^2.3.3",
     "@vitejs/plugin-vue-jsx": "^1.3.10",
     "@vue/babel-plugin-jsx": "^1.1.1",
-    "@vue/cli-plugin-unit-jest": "5.0.0-beta.2",
```

Comment:
This was a really old import that I never used, but was installing an old version of jest. So I removed it.